src/js/components/Video/doc.js

```diff
@@ -31,3 +31,36 @@ export const doc = Video => {
 
   return DocumentedVideo;
 };
+
+export const themeDoc = {
+  'dark': {
+    description: 'dark color defenition to consider when the Video renders the background color of the scrubber',
```

Comment:
"definition"

Reply:
What exactly is this? I didn't see a reference to 'dark' from a theme in any of the video files. Also, where in the theme is this? It doesn't seem like it is in `global` or `video`.

Reply:
@ericsoderberghp it's in normalizeColor() https://github.com/grommet/grommet/blob/ec03444fcd59101b8bb804d5119e29a38fd099ce/src/js/utils/colors.js#L6, I wasn't sure about it either. Its used for the video's background color